model/stats/query.go

```diff
@@ -176,23 +148,8 @@ func (f *StatsFilter) validateDates() error {
 	return catcher.Resolve()
 }
 
-// ValidateForTests validates that the StatsFilter struct is valid for use with test stats.
-func (f *StatsFilter) ValidateForTests() error {
-	catcher := grip.NewBasicCatcher()
-
-	catcher.Add(f.ValidateCommon())
-	catcher.Add(f.validateDates())
-
-	catcher.NewWhen(f.Limit > MaxQueryLimit || f.Limit <= 0, "invalid limit")
-	if f.StartAt != nil {
-		catcher.Add(f.StartAt.validateForTests(f.GroupBy))
-	}
-	catcher.NewWhen(len(f.Tests) == 0 && len(f.Tasks) == 0, "missing tests or tasks")
-
-	return catcher.Resolve()
-}
-
-// ValidateForTasks use with test stats validates that the StatsFilter struct is valid for use with task stats.
+// ValidateForTasks validates that the StatsFilter struct is valid for use with
+// task stats.
 func (f *StatsFilter) ValidateForTasks() error {
 	catcher := grip.NewBasicCatcher()
 
```

Comment:
should the `ValidateCommon`s be merged into their respective validate functions?